src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,62 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
+    public int Add(int scannetFirstNumInt, int scannetSecondNumInt, String scannerOperations) {
```

Comment:
Please use lowerCamelCase for method names https://ru.wikipedia.org/wiki/CamelCase
Also please use `static` for this methods, so it won't be needed to create instance of ConsoleCalculator